net/qpid-proton/Portfile

```diff
@@ -58,3 +63,16 @@ variant swig description {With built-in support for SWIG so the bindings can be
                             port:swig-go \
                             port:swig-ruby
 }
+
+post-build {
+    set scriptname   runtests.sh
+
+    set script [open "${worksrcpath}/../build/${scriptname}" w 0644]
```

Comment:
You should use `${build.dir}` rather than assuming its value is `${worksrcpath}/../build`.